stories/FCard.stories.ts

```diff
@@ -31,5 +43,8 @@ export const Default: StoryFn<FCardArgs> = Template.bind({});
 Default.args = {
   width: '800px',
   color: 'info',
+  hoverColor: 'info',
   text: `“Choisir Fifteen, c’est choisir la fiabilité, la sécurité et la technologie. Pour nous c’était un choix facile.”`,
+  textColor: 'neutral--light-5',
+  hoverTextColor: 'neutral--light-5',
```

Comment:
You must create a new story for the hover args. Leave this one unchanged (and you will see moreover the unwanted behavior I coined in the previous comment)